modular_skyrat/master_files/code/game/objects/items/encryptionkey.dm

```diff
@@ -84,3 +84,8 @@
 	name = "\improper Schechi translation key"
 	desc = "An encryption key that automatically encodes Schechi heard through the radio into common."
 	translated_language = /datum/language/schechi
+
+/obj/item/encryptionkey/translation/vulpkanin
+	name = "\improper Canilunzt translation key"
+	desc = "An encryption key that automatically encodes Canilunzt heard through the radio into common."
+	translated_language = /datum/language/canilunzt
```

Comment:
add a newline to prevent this 
![image](https://user-images.githubusercontent.com/41448081/228968749-45b03bc3-3757-4f29-92dd-65fa2fb55ad1.png)

Reply:
Fixed